packages/nimble-components/src/rich-text/models/tests/markdown-parser.spec.ts

```diff
@@ -344,4 +344,52 @@ describe('Markdown parser', () => {
             );
         }
     });
+
+    describe('Markdown string with hard break should have respective br tag when rendered', () => {
+        const focused: string[] = [];
+        const disabled: string[] = [];
+        const markdownStringWithHardBreak: {
+            name: string,
+            tags: string[]
+        }[] = [
+            {
+                name: '**bold**\\\n*Italics*',
+                tags: ['P', 'STRONG', 'BR', 'EM']
```

Comment:
Can we replace name here as `value` and give a valid test case title with `name` which is easy to understand?

Reply:
For ex: 
1. `should render br tag between bold and italics`
2. `should render br tag between two first level bullet list items`

Reply:
Updated